### PR TITLE
Fix exit codes when dying on a signal

### PR DESCRIPTION
**Pull request by icecrime**
Test the process `WaitStatus` for a signal, and return an exit code of [128 + signal which killed the process](http://tldp.org/LDP/abs/html/exitcodes.html).

Fixes docker/docker#9979.

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

**Comment by crosbymichael** [comment]
ping @mrunalp what do you think?

File: namespaces/exec.go [hunk 1]
```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/docker/libcontainer/system"
 )
 
+const (
+	EXIT_SIGNAL_OFFSET = 128
```
Comment (tianon) [inline namespaces/exec.go]: Why the C-style capitalization?

Comment (crosbymichael) [inline namespaces/exec.go]: Because that is how all the other consts are for signals and most of the consts in libcontainer

**Comment by mrunalp** [comment]
Looking

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**